plugin/completion.py

```diff
@@ -35,10 +37,15 @@
 SessionName = str
 CompletionResponse = Union[List[CompletionItem], CompletionList, None]
 ResolvedCompletions = Tuple[Union[CompletionResponse, Error], 'weakref.ref[Session]']
+CompletionsStore = TypedDict('CompletionsStore', {
+    'items': List[CompletionItem],
+    'itemDefaults': CompletionItemDefaults
+})
```

Comment:
Can you use a tuple here? Would simplify code a bit:

```suggestion
CompletionsStore = Tuple[List[CompletionItem], CompletionItemDefaults]
```